apps/site/src/pages/api/link-wordpress.api.ts

```diff
@@ -7,6 +7,7 @@ import { formatErrors } from "../../util/api";
 export type ApiSignupRequestBody = {
```

Comment:
Just noticed that these types have the same name as in `signup.api.ts`. I think it's worth renaming these, but it could be done in the follow up PR.